lib/node_modules/@stdlib/math/stats/anova1/lib/mean_table.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+// MODULES //
+
+var incrstdev = require( '@stdlib/math/stats/incr/stdev' );
+var uniqueElems = require('./unique_elems.js');
+
+/**
+* Computes the vector of means for each treatment.
+*
+* @param {NumericArray} x - Measured values
+* @param {Array} factor - Array of treatments
+* @returns {{}} destination array
+*/
+function meanTable(x, factor) {
+	var treatments;
+	var factorCount;
+	var meanTable;
+	var j;
+	var i;
+	var accumulator;
+	var k;
+	var newMean;
+
+
+	if (x.length !== factor.length) {
```

Comment:
It's fine to remove this check given since the function is only called inside of `make_anova.js` (where we already establish that the arrays have the same length) and won't be publicly available.

Reply:
Ok. This may be useful for future development yet for now I agree with the sentiment. Removed.